spec/support/dummy_app/config/unicorn.conf.rb

```diff
@@ -1,4 +1,4 @@
-worker_processes (ENV["UNICORN_WORKERS"] || 5).to_i
+worker_processes (ENV["UNICORN_WORKERS"] || 1).to_i
```

Comment:
Reduced number of unicorn workers to 1 to make the tests easier to reason about. It does reduce our coverage for multiple processes but I think this is a fine trade-off for now.